start-site/src/main/resources/application.yml

```diff
@@ -1020,6 +1020,21 @@ initializr:
             - rel: guide
               href: https://github.com/spring-cloud-samples/bookstore-service-broker
               description: Using Spring Cloud Open Service Broker
+        - name: App Broker
+          id: app-broker
```

Comment:
This sounds like a very generic ID to me. Given that it's open api specific, it should probably be qualified.

Reply:
is `osbapi-app-broker` better?